### PR TITLE
Seq2seq test

### PR DESCRIPTION
**Pull request by unnonouno**
I added test for seq2seq example.

### CODE REVIEW

File: test_example.sh [hunk 1]
```diff
@@ -87,6 +87,12 @@ echo "Runnig DCGAN example"
 $run examples/dcgan/train_dcgan.py -b 1 -e 1 -i ../data/dcgan --n_hidden=10 --snapshot_interval 1 --display_interval 1
 $run examples/dcgan/train_dcgan.py -b 1 -e 1 --gpu=0 -i ../data/dcgan --n_hidden=10 --snapshot_interval 1 --display_interval 1
 
+# seq2seq
+if [ -f chainer/examples/seq2seq.py ]; then
```
Comment (okuta) [inline test_example.sh]: Please remove `chainer/` . Current folder is `chainer`.

**Comment by unnonouno** [comment]
fixed

**Comment by okuta** [comment]
jenkins, test this please

File: test_example.sh [hunk 2]
```diff
@@ -87,6 +87,12 @@ echo "Runnig DCGAN example"
 $run examples/dcgan/train_dcgan.py -b 1 -e 1 -i ../data/dcgan --n_hidden=10 --snapshot_interval 1 --display_interval 1
 $run examples/dcgan/train_dcgan.py -b 1 -e 1 --gpu=0 -i ../data/dcgan --n_hidden=10 --snapshot_interval 1 --display_interval 1
 
+# seq2seq
+if [ -f examples/seq2seq.py ]; then
```
Comment (okuta) [inline test_example.sh]: `examples/seq2seq/seq2seq.py` ?

Comment (unnonouno) [reply test_example.sh]: oh

**Comment by unnonouno** [comment]
jenkins test this please

**Comment by unnonouno** [comment]
jenkins test this please

**Comment by okuta** [comment]
LGTM!

**Pull request merged**